package.json

```diff
@@ -92,7 +92,7 @@
   },
   "version": "0.0.0",
   "resolutions": {
-    "@types/react": "16.14.0",
+    "@types/react": "17.0.2",
```

Comment:
Do you know why we need this one? If cli-kit is overriding this value, which package needs it? Couldn't we move it to where it's needed?

Reply:
No luck, I think the tree of dependencies doesn't manage to pick up the specific version in the packages 🤷